examples/field_examples/viking_graben_line12/download_data.ipynb

```diff
@@ -0,0 +1,146 @@
+{
```

Comment:
Would make just a bash script for something like that

Reply:
ok